homeassistant/components/tractive/config_flow.py

```diff
@@ -0,0 +1,74 @@
+"""Config flow for tractive integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import aiotractive
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({CONF_EMAIL: str, CONF_PASSWORD: str})
```

Comment:
It doesn't look like email and password are optional. Please mark them as `vol.Required`.